Astro Release Notes (June 9th) (#792)
astronomer/docs: astro/api-keys.md

Deployment API keys are primarily used to automate actions that otherwise require manual inputs. They allow you to:

- Deploy code to Astro [using CI/CD](ci-cd.md) with tools such as GitHub Actions or Jenkins.
- Deploy code and configuration changes to Astro [using the Astro CLI](deploy-code.md) without user authentication.
- Automate requests to the [Airflow REST API](airflow-api.md).

To use API keys with the Astro CLI, you need to make your Deployment API key ID and secret accessible to the machine running the CLI via the following OS-level environment variables:

- `ASTRONOMER_KEY_ID`
- `ASTRONOMER_KEY_SECRET`

> For example, to update a given Deployment using the Astro CLI on a Mac machine, you can set temporary OS-level environment variables using the following command:

Author: ```suggestion
For example, to update a given Deployment using the Astro CLI on a Mac machine, set temporary OS-level environment variables with the following commands:
```

```sh
export ASTRONOMER_KEY_ID=<your-key-id>
export ASTRONOMER_KEY_SECRET=<your-key-id>
```

> After setting the variables, running `astro deployment update` works for the Deployment without you having to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.

Reviewer: ```suggestion
After setting the variables, running `astro deployment update` works for the Deployment and you don't need to manually authenticate to Astronomer. Astronomer recommends storing `ASTRONOMER_KEY_SECRET` as a secret before using it to programmatically update production-level Deployments.
```
